modules/runtime/pages/process-manager-application.adoc

```diff
@@ -0,0 +1,60 @@
+= Creating a process manager application
+:description: This page explains how to create your own custom process manager application
+
+[NOTE]
+====
+For Subscription editions only. +
+If your Bonita Subscription has been purchased before 2021, this Add-on can be purchased on top of your subscription.
```

Comment:
Why do you talk about BCD here?? This page doesn't have anything to do with BCD.

Reply:
I think it was a copy paste from the first result that I got, my mistake

Reply:
Just put 'For Enterprise, Performance and Efficiency editions only.' ;)